src/templates/finops-hub/modules/dataExplorer.bicep

```diff
@@ -0,0 +1,134 @@
+//==============================================================================
+// Parameters
+//==============================================================================
+
+@description('Required. Name of the hub. Used to ensure unique resource names.')
+param hubName string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Resource tags.')
+param tags object = {}
+
+@description('Required. The name of the Azure Key Vault instance.')
+param keyVaultName string
+
+//------------------------------------------------------------------------------
+// Variables
+//------------------------------------------------------------------------------
+
+// Generate globally unique cluster name: 4-22 chars; lowercase letters and numbers
+var safeHubName = replace(replace(toLower(hubName), '-', ''), '_', '')
+var clusterName = replace('${take(safeHubName, 22)}', '--', '-')
+
+//==============================================================================
+// Resources
+//==============================================================================
+
+resource cluster 'Microsoft.Kusto/clusters@2022-12-29' = {
+  name: clusterName
+  location: location
+  tags: tags
+  sku: {
+    capacity: 1
+    name: 'string'
+    tier: 'string'
+  }
+  identity: {
```

Comment:
Do we need this? If so, what do we need to set?

Reply:
You only need these if you want to use Managed Identities (MIs). On behalf of who will the ingestions into Kusto run? If you plan to use managed identities for this, then you'll need to put the MI details here.

Reply:
You tell me 🙂 What's ideal? I'm assuming we probably do want a managed identity to run the ingestion. I don't know enough to make that call, tho. Thoughts? If we do it, can you share what this should be set to?

Reply:
Let's talk about it.